Shared/Samples/Download vector tiles to local cache/DownloadVectorTilesToLocalCacheView.swift

```diff
@@ -278,6 +287,14 @@ private extension DownloadVectorTilesToLocalCacheView {
             try? FileManager.default.removeItem(at: styleTemporaryURL)
         }
         
+        /// Updates the export's max scale.
+        func updateMaxScale(_ maxScale: Double) {
```

Comment:
Do you need a doc comment for the `maxScale` parameter here?

Reply:
I adjusted the logic here to get the scale from the view.